audio.py

```diff
@@ -70,6 +116,9 @@ def read_audio(
 	if sample_rate_ != sample_rate:
 		signal, sample_rate_ = resample(signal, sample_rate_, sample_rate)
 
+	if show_time:
```

Comment:
I have a better idea now. Let's instead add a decorator in metrics.py:
```
import functools
def measure_time(func):
  @functools.wraps(func) 
  def wrapped(*args, **kwargs):
    return func(*args, **kwargs)
  return wrapped 
```

Reply:
done